.github/workflows/tests-ipsec-upgrade.yaml

```diff
@@ -78,7 +78,23 @@ jobs:
         with:
           sha: ${{ inputs.SHA || github.sha }}
 
+  wait-for-images:
+    name: Wait for images
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+      - name: Checkout context ref (trusted)
+        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+        with:
+          ref: ${{ inputs.context-ref || github.sha }}
+          persist-credentials: false
+      - name: Wait for images
```

Comment:
hm, I got confused as that wait step has:
```
if: ${{ steps.vars.outputs.downgrade_version != '' }}
```
but it waits for regular images "steps.vars.outputs.sha". That has to be a bug, no :thinking: ?

Reply:
AFAIU, that `if` condition is because all the steps are skipped when downgrade_version is empty (that is, e.g., for patch upgrade/downgrade on main). But the step is actually intended to wait for the regular images.

Reply:
ah yea, makes sense, fixed.